mautic/mautic
# Outlook Button UI fix in email builder

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
As the Outlook filter out padding, the button will not be same as design we build in email builder . This PR will replace it with table element to create an button. So the design will be as expected.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create New Template Email in email section
2. Select one the and use the builder to add button
3. Send the email and check the mail in outlook (2010) 

#### Steps to test this PR:
1. Apply PR
2. Create New Template Email in email section
3. Select one the and use the builder to add button
4. Send the email and check the mail in outlook (2010)
